src/window/keyboard_manager.rs

```diff
@@ -56,10 +56,13 @@ impl KeyboardManager {
                 log::trace!("Ime commit {text}");
                 send_ui(SerialCommand::Keyboard(text.to_string()));
             }
-            Event::WindowEvent {
-                event: WindowEvent::Ime(Ime::Preedit(text, cursor_offset)),
-                ..
-            } => self.ime_preedit = (text.to_string(), *cursor_offset),
+            // Event::WindowEvent {
+            //     event: WindowEvent::Ime(Ime::Preedit(text, cursor_offset)),
+            //     ..
+            // } => {
+            //     self.ime_preedit = (text.to_string(), *cursor_offset);
+            //     log::trace!("Ime preedit {text}");
+            // }
```

Comment:
I would prefer to keep the ime state in the keyboard manager. And I also think that this breaks something, since we used to block keyboard inputs when the pre-edit was active, and that code is no longer effective.